Refresh around dependencies #15750 (#16190)
github/docs: content/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository.md

## Changing the "Used by" package

> You may notice some repositories have a "Used by" section in the sidebar of their **Code** tab. In order for your repository to display this "Used by" section, three conditions must be met:

Reviewer: We try to avoid using pronouns such as `their` for localization purposes as they may be ambiguous + simplifying:
```suggestion
You may notice some repositories have a "Used by" section in the sidebar of the **Code** tab. Your repository will have a "Used by" section if:
```

  * The dependency graph is enabled for the repository (see the above section for more details).
  * Your repository contains a package that is published on a [supported package ecosystem](/github/visualizing-repository-data-with-graphs/about-the-dependency-graph#supported-package-ecosystems).

>   * In said ecosystem, your package has a link to a *public* repository where the source is stored.

Reviewer: I think we use italics to emphasize something, also removing `said` as it can be misinterpreted during translation
```suggestion
  * Within the ecosystem, your package has a link to a _public_ repository where the source is stored.
```
